lib/Doctrine/Common/Cache/ChainCache.php

```diff
@@ -15,6 +15,9 @@ class ChainCache extends CacheProvider
     /** @var CacheProvider[] */
     private $cacheProviders = [];
 
+    /** @var int */
+    private $defaultLifeTimeForDownstreamCacheProviders;
```

Comment:
This would override the default value `0` of `doSave()` with `null`. What if we keep the default value in here too?
```suggestion
    private $defaultLifeTimeForDownstreamCacheProviders = 0;
```

Reply:
Ah, I missed that default, totally true!